CanonicalLtd/snappy-docs
# Dockerfile for building production docker image

**NB: Review #107 first**

This dockerfile will build the image we will use in production for docs.snapcraft.io.

QA
--

``` bash
./run build
docker build --tag snappy-docs .
docker run -ti -p 80:80 snappy-docs
```

Now go to <http://localhost:80> and check the site looks and behaves just like <https://snapcraft.io/docs>.
